styles/globals.css

```diff
@@ -1,3 +0,0 @@
-@tailwind base;
```

Comment:
Muzes pls toto nechat a misto importu _output.css importovat globals.css? V nextu je konvence mit styles/globals.css (a na to upozornuje i setup article tady (https://tailwindcss.com/docs/guides/nextjs)